src/tinyuf2_board.c

```diff
@@ -13,31 +13,54 @@
 #include "ice_flash.h"
 #include "ice_fpga.h"
 #include "ice_spi.h"
+#include "ice_sram.h"
 
 static alarm_id_t alarm_id;
-static bool flash_ready;
+static bool spi_ready;
+static uint32_t flash_erased[ICE_FLASH_SIZE_BYTES / (ICE_FLASH_BLOCK_SIZE * 32)];
```

Comment:
```
ICE_FLASH_SIZE_BYTES=$((4 * 1024 * 1024))
ICE_FLASH_BLOCK_SIZE=65536
echo $((ICE_FLASH_SIZE_BYTES / (ICE_FLASH_BLOCK_SIZE * 32)))
```

This gives us `flash_erased[2]`... Kind of small... Still worth it?
Oh yes, this is a *bitmap* yes it is worth it!